docs/core/porting/cpp-cli.md

```diff
@@ -1,52 +1,57 @@
 ---
-title: Migrate C++/CLI projects to .NET Core and .NET 5+
-description: Learn about porting C++/CLI projects to .NET Core and .NET 5 and later versions.
+title: Migrate C++/CLI projects to .NET
+description: Learn about porting C++/CLI projects to .NET.
 author: mjrousos
-ms.date: 10/27/2023
+ms.date: 01/23/2024
 ---
 
-# How to port a C++/CLI project to .NET Core or .NET 5+
+# How to port a C++/CLI project to .NET
 
-Beginning with .NET Core 3.1 and Visual Studio 2019, [C++/CLI projects](/cpp/dotnet/dotnet-programming-with-cpp-cli-visual-cpp) can target .NET Core. This support makes it possible to port Windows desktop applications with C++/CLI interop layers to .NET Core/.NET 5+. This article describes how to port C++/CLI projects from .NET Framework to .NET Core 3.1.
+Beginning with Visual Studio 2019, [C++/CLI projects](/cpp/dotnet/dotnet-programming-with-cpp-cli-visual-cpp) can target .NET. This support makes it possible to port Windows desktop applications with C++/CLI interop layers to .NET. This article describes how to port C++/CLI projects from .NET Framework to .NET.
 
 ## C++/CLI .NET Core limitations
 
-There are some important limitations to porting C++/CLI projects to .NET Core compared to other languages:
+There are some important with C++/CLI projects and .NET compared to .NET Framework:
 
-- C++/CLI support for .NET Core is Windows only.
-- C++/CLI projects can't target .NET Standard, only .NET Core or .NET Framework.
-- C++/CLI projects don't support the new SDK-style project file format. Instead, even when targeting .NET Core, C++/CLI projects use the existing *.vcxproj* file format.
-- C++/CLI projects can't multitarget multiple .NET platforms. If you need to build a C++/CLI project for both .NET Framework and .NET Core, use separate project files.
-- .NET Core doesn't support `-clr:pure` or `-clr:safe` compilation, only the new `-clr:netcore` option (which is equivalent to `-clr` for .NET Framework).
+- C++/CLI support for .NET is Windows only.
+- C++/CLI projects can't target .NET Standard.
+- C++/CLI projects don't support the new SDK-style project file format. Instead, C++/CLI projects use the existing _.vcxproj_ file format.
+- C++/CLI projects can't target multiple .NET platforms. If you need to build a C++/CLI project for both .NET and .NET Framework.
```

Comment:
```suggestion
- C++/CLI projects can't target multiple .NET platforms. If you need to build a C++/CLI project for both .NET and .NET Framework, use separate project files.
```

Reply:
Same comment for "C++/CLI projects don't support the new SDK-style project file format. Instead, C++/CLI projects use the existing _.vcxproj_ file format."